README.md

```diff
@@ -21,7 +21,7 @@ entire Chrome OS desktop, or to inspect a specific browser tab.
 This tool is mainly useful for the following types of people:
 - Developers/QA working on Chrome accessibility support
 - Developers/QA working on assistive technologies for Chrome OS
-- Developers/QA working on accessibility of web applications looking (less so)
+- Developers/QA working on accessibility of web applications
```

Comment:
The rationale behind this line is that saying some group of people are less likely to do something might indeed discourage them to do that... 🤔